device-simulator/Dockerfile

```diff
@@ -1,10 +1,14 @@
-FROM alpine:3.14 AS build
+FROM ubuntu:20.04 AS service
 
-RUN apk add --no-cache curl git build-base gcc linux-headers patch
+RUN apt-get update -y && \
+ apt-get install -y bash curl gdb git-core g++ ca-certificates patch make --no-install-recommends
 RUN git clone --recursive -b plgd.dev https://github.com/iotivity/iotivity-lite.git
 COPY ./src/ /iotivity-lite/
-RUN make -C /iotivity-lite/port/linux CLOUD=1 SECURE=1 DEBUG=1 MNT=1 cloud_server
-
-FROM alpine:3.14 AS service
-COPY --from=build /iotivity-lite/port/linux/cloud_server /usr/local/bin/cloud_server
-ENTRYPOINT ["/usr/local/bin/cloud_server"]
+RUN make -C /iotivity-lite/port/linux CLOUD=1 SECURE=1 OSCORE=0 DEBUG=1 MNT=1 ASAN=0 cloud_server
+#RUN curl -sSfL https://github.com/mapbox/logbt/archive/v3.0.0.tar.gz | tar --gunzip --extract --strip-components=1 --exclude="*md" --exclude="test*" --directory=/usr/local
+COPY logbt /usr/local/bin/logbt
```

Comment:
where does logbt binary come from if line 8 is commented out?

Reply:
It's shell script and it is stored in device-simulator direcotry